test/tools/isoltest.cpp

```diff
@@ -242,8 +246,31 @@ TestStats TestTool::processPath(
 
 }
 
+/// Ensures the terminal is set up properly.
+void setupTerminal()
+{
+#if defined(_WIN32)
+	// set output mode to handle virtual terminal (ANSI escape sequences)
```

Comment:
Please start with an uppercase word and end with a `.` if you use full sentences :)